client/views/question.html

```diff
@@ -1,44 +1,101 @@
-<div id="question-container-inner">
-    <!-- Minimum Player Required Popup-->
-     <div style="background-color: teal; height: 40px; width: 400px" ng-show="game.errorPlayerMin">{{game.errorPlayerMin}}</div>
-     <!-- Player Invite Success Popup-->
-     <span>
-         <div style="background-color: teal; height: 40px; width: 400px" ng-show="game.inviteMessage">{{game.inviteMessage}}</div>
-       </span>
-  <div class="card longBlack">
-    <span ng-bind-html-unsafe="game.curQuestion.text" ng-hide="game.state === 'game ended' || game.state ==='game dissolved' || game.state === 'awaiting players'"></span>
+<!-- Minimum Player Required Popup-->
+<div
+  style="background-color: teal; height: 40px; width: 400px"
+  ng-show="game.errorPlayerMin"
+>
+  {{game.errorPlayerMin}}
+</div>
 
-    <div id="notifications" ng-if="game.notification" ng-animate="{enter:'animated fadeInLeft', leave:'animated fadeOut'}">{{game.notification}}</div>
+<!-- Player Invite Success Popup-->
+<span>
+  <div
+    style="background-color: teal; height: 40px; width: 400px"
```

Comment:
The team agreed to avoid inline styles. Kindly extract this style into a class in a css file.

Reply:
Thanks for the feedback. Will make corrections and ensure I don't make such mistakes in future.

Reply:
Fixed!